src/Microsoft.ML.Data/DataView/InternalSchemaDefinition.cs

```diff
@@ -147,11 +147,11 @@ public static void GetVectorAndItemType(MemberInfo memberInfo, out bool isVector
             switch (memberInfo)
             {
                 case FieldInfo fieldInfo:
-                    GetVectorAndItemType(fieldInfo.FieldType, fieldInfo.Name, out isVector, out itemType);
+                    GetVectorAndItemType(fieldInfo.FieldType, fieldInfo.Name, out isVector, out itemType, fieldInfo.GetCustomAttributes().ToArray());
```

Comment:
GetCustomAttributes already returns an array - why the need to call `ToArray()` again?

Reply:
It's not an array. It's an IEnumerable. Anyway, I change `params` to array.

---
In reply to: [286691422](https://github.com/dotnet/machinelearning/pull/3263#discussion_r286691422) [](ancestors = 286691422)